datus-redshift/datus_redshift/connector.py

```
@@ -0,0 +1,1265 @@
+    def _sys_schemas(self) -> Set[str]:
+        """
+        Return set of system schemas to filter out from listings.
+        
+        These are Redshift system schemas that should be hidden from users.
+        
+        Returns:
+            Set of system schema names
+        """
+        return {"pg_catalog", "information_schema", "pg_internal"}
+
+    def do_switch_context(self, catalog_name: str = "", database_name: str = "", schema_name: str = ""):
```

Ruff (0.14.5)

Unused method argument: `catalog_name`

(ARG002)